src/App.tsx

```diff
@@ -26,42 +27,44 @@ function App() {
     <div className="App">
       <BrowserRouter>
         <CookiesProvider>
-          <FeatureFlagProvider>
-            <AuthProvider>
-              <Routes>
-                <Route element={<PageLayout />}>
-                  <Route path="/legal/privacy" element={<PrivacyPolicy />} />
-                  <Route path="/legal/terms" element={<TermsOfUse />} />
-                </Route>
-                <Route path="/login" element={<Login />} />
-                <Route path="/v1">
-                  <Route element={<Layout />}>
+          <AppProgress.Provider>
+            <FeatureFlagProvider>
+              <AuthProvider>
+                <Routes>
+                  <Route element={<PageLayout />}>
+                    <Route path="/legal/privacy" element={<PrivacyPolicy />} />
+                    <Route path="/legal/terms" element={<TermsOfUse />} />
+                  </Route>
+                  <Route path="/login" element={<Login />} />
+                  <Route path="/v1">
+                    <Route element={<Layout />}>
+                      <Route
+                        path="stories/:action"
+                        element={
+                          <RequireAuth>
+                            <StoryGeneratorV1 />
+                          </RequireAuth>
+                        }
+                      />
+                      <Route index element={<Navigate to="stories/new" />} />
+                    </Route>
+                  </Route>
+                  <Route path="/v0">
                     <Route
                       path="stories/:action"
                       element={
                         <RequireAuth>
-                          <StoryGeneratorV1 />
```

Comment:
Why are we removing the version?

Reply:
We just moved the route for `V1` above that for `V0`